lib/pure/unicode.nim

```diff
@@ -323,970 +336,970 @@ proc runeSubStr*(s: string, pos:int, len:int = int.high): string =
-    0x0b5c,  0x0b5d,  #  -
-    0x0b5f,  0x0b61,  #  -
-    0x0b85,  0x0b8a,  #  -
-    0x0b8e,  0x0b90,  #  -
-    0x0b92,  0x0b95,  #  -
-    0x0b99,  0x0b9a,  #  -
-    0x0b9e,  0x0b9f,  #  -
-    0x0ba3,  0x0ba4,  #  -
-    0x0ba8,  0x0baa,  #  -
-    0x0bae,  0x0bb5,  #  -
-    0x0bb7,  0x0bb9,  #  -
-    0x0c05,  0x0c0c,  #  -
-    0x0c0e,  0x0c10,  #  -
-    0x0c12,  0x0c28,  #  -
-    0x0c2a,  0x0c33,  #  -
-    0x0c35,  0x0c39,  #  -
-    0x0c60,  0x0c61,  #  -
-    0x0c85,  0x0c8c,  #  -
-    0x0c8e,  0x0c90,  #  -
-    0x0c92,  0x0ca8,  #  -
-    0x0caa,  0x0cb3,  #  -
-    0x0cb5,  0x0cb9,  #  -
-    0x0ce0,  0x0ce1,  #  -
-    0x0d05,  0x0d0c,  #  -
-    0x0d0e,  0x0d10,  #  -
-    0x0d12,  0x0d28,  #  -
-    0x0d2a,  0x0d39,  #  -
-    0x0d60,  0x0d61,  #  -
-    0x0e01,  0x0e30,  #  -
-    0x0e32,  0x0e33,  #  -
-    0x0e40,  0x0e46,  #  -
-    0x0e5a,  0x0e5b,  #  -
-    0x0e81,  0x0e82,  #  -
-    0x0e87,  0x0e88,  #  -
-    0x0e94,  0x0e97,  #  -
-    0x0e99,  0x0e9f,  #  -
-    0x0ea1,  0x0ea3,  #  -
-    0x0eaa,  0x0eab,  #  -
-    0x0ead,  0x0eae,  #  -
-    0x0eb2,  0x0eb3,  #  -
-    0x0ec0,  0x0ec4,  #  -
-    0x0edc,  0x0edd,  #  -
-    0x0f18,  0x0f19,  #  -
-    0x0f40,  0x0f47,  #  -
-    0x0f49,  0x0f69,  #  -
-    0x10d0,  0x10f6,  #  -
-    0x1100,  0x1159,  #  -
-    0x115f,  0x11a2,  #  -
-    0x11a8,  0x11f9,  #  -
-    0x1e00,  0x1e9b,  #  -
-    0x1f50,  0x1f57,  #  -
-    0x1f80,  0x1fb4,  #  -
-    0x1fb6,  0x1fbc,  #  -
-    0x1fc2,  0x1fc4,  #  -
-    0x1fc6,  0x1fcc,  #  -
-    0x1fd0,  0x1fd3,  #  -
-    0x1fd6,  0x1fdb,  #  -
-    0x1fe0,  0x1fec,  #  -
-    0x1ff2,  0x1ff4,  #  -
-    0x1ff6,  0x1ffc,  #  -
-    0x210a,  0x2113,  #  -
-    0x2115,  0x211d,  #  -
-    0x2120,  0x2122,  #  -
-    0x212a,  0x2131,  #  -
-    0x2133,  0x2138,  #  -
-    0x3041,  0x3094,  #  -
-    0x30a1,  0x30fa,  #  -
-    0x3105,  0x312c,  #  -
-    0x3131,  0x318e,  #  -
-    0x3192,  0x319f,  #  -
-    0x3260,  0x327b,  #  -
-    0x328a,  0x32b0,  #  -
-    0x32d0,  0x32fe,  #  -
-    0x3300,  0x3357,  #  -
-    0x3371,  0x3376,  #  -
-    0x337b,  0x3394,  #  -
-    0x3399,  0x339e,  #  -
-    0x33a9,  0x33ad,  #  -
-    0x33b0,  0x33c1,  #  -
-    0x33c3,  0x33c5,  #  -
-    0x33c7,  0x33d7,  #  -
-    0x33d9,  0x33dd,  #  -
-    0x4e00,  0x9fff,  #  -
-    0xac00,  0xd7a3,  #  -
-    0xf900,  0xfb06,  #  -
-    0xfb13,  0xfb17,  #  -
-    0xfb1f,  0xfb28,  #  -
-    0xfb2a,  0xfb36,  #  -
-    0xfb38,  0xfb3c,  #  -
-    0xfb40,  0xfb41,  #  -
-    0xfb43,  0xfb44,  #  -
-    0xfb46,  0xfbb1,  #  -
-    0xfbd3,  0xfd3d,  #  -
-    0xfd50,  0xfd8f,  #  -
-    0xfd92,  0xfdc7,  #  -
-    0xfdf0,  0xfdf9,  #  -
-    0xfe70,  0xfe72,  #  -
-    0xfe76,  0xfefc,  #  -
-    0xff66,  0xff6f,  #  -
-    0xff71,  0xff9d,  #  -
-    0xffa0,  0xffbe,  #  -
-    0xffc2,  0xffc7,  #  -
-    0xffca,  0xffcf,  #  -
-    0xffd2,  0xffd7,  #  -
-    0xffda,  0xffdc]  #  -
+    0x00d8.Rune,  0x00f6.Rune,  #  -
```

Comment:
😭 

This seriously calls for a macro. We should implement a ``asArray[Rune]`` macro that does this for us (or a better name). I'm going to create an issue for this and maybe even implement it myself (perhaps even as part of a livestream :)

Reply:
https://github.com/nim-lang/Nim/issues/6563